packages/datadog-instrumentations/src/rhea.js

```diff
@@ -167,7 +167,8 @@ function wrapDeliveryUpdate (obj, update) {
 function patchCircularBuffer (proto, Session) {
   Object.defineProperty(proto, 'outgoing', {
     configurable: true,
-    get () { return undefined },
+    // eslint-disable-next-line getter-return
```

Comment:
Why not just keep the explicit undefined return instead of having to add a comment?

Reply:
Shouldn't we try to avoid conflicting rules? Which rule is conflicting?